src/digger/digger/digger_node.py

```diff
@@ -101,17 +90,15 @@ def toggle(self, digger_belt_power: float) -> None:
         else:
             self.set_power(digger_belt_power)
 
-    # TODO: This method can probably be deleted during the implementation of ticket #257
     def set_position(self, position: int) -> None:
         """This method sets the position (in degrees) of the digger."""
-        self.current_goal_position = position  # goal position should be in degrees
         self.cli_motor_set.call_async(
             MotorCommandSet.Request(
```

Comment:
```suggestion
await MotorCommandSet.Request(
```

Reply:
you can't await here, at least according to my VS code, because this function is not async

Reply:
You need to make the function async

Reply:
Instead of this, what about wrapping the whole commandset call in a rclpy spin_until_future_complete(DiggerNode, MotorCommandSet(...))

Reply:
I committed the change here's more explanation of why that isn't a great idea.
https://github.com/ros2/rclpy/issues/1313#issuecomment-2307615945